mysql-test/include/have_rocksdb_as_default.inc

```diff
@@ -0,0 +1,18 @@
+--source include/have_rocksdb.inc
+
+if (`SELECT COUNT(*) = 0 FROM INFORMATION_SCHEMA.ENGINES WHERE engine = 'rocksdb' AND support IN ('DEFAULT')`)
+{
+  --skip Test requires default engine RocksDB
+}
+
+# MYR-14
```

Comment:
OK. Would be good to track somewhere the smaller items that need to be resolved on the experimental -> RC transition for MYR markers and the like